recipe/meta.yaml

```
@@ -92,6 +93,9 @@ outputs:
         - __cuda >={{ cuda_compiler_version_min }}  # [cuda_compiler_version != "None"]
       run_constrained:
         - apache-arrow-proc =*={{ build_ext }}
+        # keep lower pin aligned with run_exports from numpy
+        # https://github.com/conda-forge/numpy-feedstock/blob/main/recipe/meta.yaml
+        - {{ pin_compatible("numpy", lower_bound="1.21", max_pin="x") }}
```

Nevermind, conda-build gets this wrong when it crosses more than a major version (the v1 recipe format will do better). We'll have to do it fully manually
```suggestion
        - numpy >=1.21,<3
```